docs/source/concepts/deploy.rst

```diff
@@ -6,15 +6,73 @@ Deploying Bento
 Deployment Overview
 -------------------
 
-The three most common deployment options with BentoML are:
-
-- 🐳 Generate container images from Bento for custom docker deployment
-- `🦄️ Yatai <https://github.com/bentoml/Yatai>`_: Model Deployment at scale on Kubernetes
-- `🚀 bentoctl <https://github.com/bentoml/bentoctl>`_: Fast model deployment on any cloud platform
-
-
-Containerize Bentos
--------------------
+Here are the four most common deployment options with BentoML:
+
+.. list-table::
+   :header-rows: 1
+
+   * - Deploy Method 
+     - Pros ❤️ 
+     - Cons 💔 
+   * - `🍱 BentoCloud <https://www.bentoml.com/>`_
+     - Serverless cloud for AI, fully managed, made for BentoML
+     - Limited availability, `sign up here <https://www.bentoml.com/bento-cloud/>`_ today for early access
```

Comment:
This might be mis-interpreted as low-availability. Let's just mention it's invite only today.